packages/remix/src/build.ts

```diff
@@ -209,15 +214,34 @@ export const build: BuildV2 = async ({
         REMIX_RUN_DEV_MAX_VERSION,
         remixVersion
       );
-      depsToAdd.push(`@vercel/remix@${vercelRemixVersion}`);
+      pkg.dependencies['@vercel/remix'] = vercelRemixVersion;
+      depsModified = true;
     }
   }
 
-  if (depsToAdd.length) {
-    await addDependencies(cliType, depsToAdd, {
-      ...spawnOpts,
-      cwd: entrypointFsDirname,
-    });
+  if (depsModified) {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be. -->
**suggestion (non-blocking):** ‏Consider adding a `debug()` log mentioning the `package.json` is being updated.